### PR TITLE
ZCS-750: Universal UI: Calendar day view

### PR DESCRIPTION
**Pull request by tarangkhandelwal**
Changeset:
* ZmCalDayView.js:
	* Overriding ZmCalColView.prototype.fbStatusBarEnabled to return false to have no fb view container along side hour divs.
	* Changing Split/Merge button experience. Now have to different buttons visible at once for split & merge operations.
	* Removing space around different calendars and merging the views end to end.

* ZmCalMonthView.js: Adding right border for weekview container.
* Other changes for redesigining appointment widget & day view fit & finish as per the UX designs.

### CODE REVIEW

File: WebRoot/css/zm.css [hunk 1]
```diff
@@ -2613,6 +2614,27 @@ INPUT[type="text"].InlineWidget {
...
+div[name='_toggleBtnContainerId'] {
+	@TopSeparator@
+	@RightSeparator@
+	@WidgetHeaderContainer-normal@
+	@Text-uppercase@
+}
+
+.ZCalToggleBtn {
+	height:2rem;
+	color:@@SecondaryBrandingColor@;
```
Comment (kalpeshhpatel) [inline WebRoot/css/zm.css]: Extra `@`

Comment (kalpeshhpatel) [reply WebRoot/css/zm.css]: Fixed: https://github.com/Zimbra/zm-web-client/pull/145/files#diff-33f9c57f22cd743824b2918132c5d73bR2627

File: WebRoot/css/zm.css [hunk 2]
```diff
@@ -2899,121 +2925,114 @@ td.calendar_month_day_item table td {
...
 }
 
-.appt_30_body TABLE,
-.appt_allday_body TABLE {
-	
+HTML>BODY .appt-selected .appt_body TD.appt_30_name,
+HTML>BODY .appt-selected .appt_allday_body TD.appt_30_name,
+HTML>BODY .appt-selected .appt_30_body TD.appt_30_name {
+	#IFDEF GECKO
+		-moz-user-select:none;
```
Comment (silentsakky) [inline WebRoot/css/zm.css]: i guess there is already a property to handle this in skin.properties

File: WebRoot/skins/_base/base4/skin.properties [hunk 3]
```diff
@@ -2599,3 +2601,26 @@ ApptEditLocationConflictColor       = @red@
 ApptEditLocationResolvedColor       = @green@
 ApptEditLocationStatusConflict      = color:@ApptEditLocationConflictColor@;
 ApptEditLocationStatusResolved      = color:@ApptEditLocationResolvedColor@;
+
+######################################
+# Appointment widget style
+######################################
+
+Appt                = z-index:10; @FixBoxModel@; @MediumRoundCorners@; overflow:hidden; opacity:0.9;
+ApptSelected        = @Appt@; z-index:11; outline: none; border:none; box-shadow: 2px 2px 2px 0 #777, 0px 0px 1px 0px #333; opacity:1;
```
Comment (silentsakky) [inline WebRoot/skins/_base/base4/skin.properties]: do we have color variables for #777 and #333?

Comment (kalpeshhpatel) [reply WebRoot/skins/_base/base4/skin.properties]: No we don't have. We do have use of this values to some property, but those are not related at all.

File: WebRoot/css/zm.css (hunk 2 above)
Comment (kalpeshhpatel) [reply WebRoot/css/zm.css]: Noted and updated here: https://github.com/Zimbra/zm-web-client/pull/145/commits/1ba1c92bb0a7e585b2f06c03988eae7ccf0a1f84

Thanks

**Pull request merged**